terraform/region/kms.tf

```diff
@@ -1,3 +1,53 @@
+resource "aws_kms_key" "multi_region_pdf_sqs_encryption_key" {
+  enable_key_rotation = true
```

Comment:
What's the key rotation period for this?

Reply:
the key rotation is defined in this document. https://docs.aws.amazon.com/kms/latest/developerguide/rotate-keys.html - it's 365 days, this should be seamless, and more than adequate for the needs of the pdf queue.